tests/libnet/job/job.go

```diff
@@ -139,10 +138,14 @@ func WaitForJob(job *batchv1.Job, toSucceed bool, timeout time.Duration) error {
 				break
 			case batchv1.JobFailureTarget:
 				break
+			case batchv1.JobSuccessCriteriaMet:
+				break
```

Comment:
Can we use `default:` instead of all these cases?

Reply:
TBH I'm not sure what should be the default behavior. That's why I preferred to keep it case specific, as was done before

Reply:
Umm maybe but i think we can aggregate it to a single case something like
`    case "apple", "banana", "orange":`

Reply:
[DONE](https://github.com/kubevirt/kubevirt/compare/5fd9d33e244eae0c69afbb68bee413ecf94965d0..4c1820d066363634fcf46f8c774d91c42f391509)

Reply:
golang do not fallthrough through cases if not explicit set. So as is now:
			`case batchv1.JobSuspended`: `<do not break>`
			`case batchv1.JobFailureTarget`: `<do not break>`
			`case batchv1.JobSuccessCriteriaMet`: `<it breaks>`
Everything works the same here since a single break works like an empty statement, but keep in mind that right now, it is working not because of an hypotetical fallthrough.
Also consider that, from https://go.dev/ref/spec#Break_statements, the "break" statement terminates execution of the innermost ["for"](https://go.dev/ref/spec#For_statements), ["switch"](https://go.dev/ref/spec#Switch_statements), or ["select"](https://go.dev/ref/spec#Select_statements) statement within the same function.
IOW we can completely drop the last 3 cases.